### PR TITLE
Issue #3481: CKEditor: Select image library is disabled

### PR DESCRIPTION
**Pull request by opi**

### CODE REVIEW

File: core/modules/filter/js/filter.js [hunk 1]
```diff
@@ -143,8 +143,8 @@ Backdrop.behaviors.editorImageDialog = {
 
     // Initialize styles of Dialog.
     if ($newToggles.length) {
-      // Initialize the Insert Link dialog on load.
-      $(".form-type-managed-file").css({ "display": "none"});
+      // Disable the Link "Upload" dialog on load.
```
Comment (klonos) [inline core/modules/filter/js/filter.js]:
```suggestion
      // Disable the "Upload a file and link to it" dialog on load.
```

**Comment by opi** [comment]
new sandbox

File: core/modules/filter/js/filter.js [hunk 2]
```diff
@@ -260,8 +258,8 @@ Backdrop.behaviors.editorImageDialog = {
       }
     });
 
-    // Add a very short delay to allow the dialog to appear.
-    window.setTimeout(function() {
+    // Listen to the dialog creation event.
```
Comment (klonos) [inline core/modules/filter/js/filter.js]:
```suggestion
    // Listen for the dialog creation event.
```

😄

**Comment by quicksketch** [comment]
Merged into 1.x and 1.12.x. Thanks!

**Pull request merged**